### PR TITLE
Updating the README

### PR DESCRIPTION
**Pull request by aimalz**
The README should give potential users/developers the big picture of RAIL and needs to be fleshed out.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,9 +3,10 @@
 
 # RAIL: Redshift Assessment Infrastructure Layers
 
-This repo is home to a series of LSST-DESC projects aiming to quantify the impact of imperfect prior information on probabilistic redshift estimation.
-RAIL differs from [PZIncomplete](https://github.com/LSSTDESC/pz_incomplete) in that it is broken into stages, each corresponding to a manageable unit of infrastructure advancement, a specific question, and a potential publication opportunity.
-By pursuing the piecemeal development of RAIL, we aim to achieve the broad goals of PZIncomplete.
+This repo is home to the back-end of the LSST-DESC PZ WG's software pipelines aiming to guide the selection and implementation of redshift estimators, encompassing the former [PZIncomplete](https://github.com/LSSTDESC/pz_incomplete) and [PZCalibrate](https://github.com/LSSTDESC/pz_calibrate) codes.
```
Comment (johannct) [inline README.md]: so, we remove any link to these repos that will not provide any further information to users, beyond a vague recollection of things past?

File: README.md [hunk 2]
```diff
@@ -3,9 +3,10 @@
...
+RAIL differs from PZIncomplete in that it is broken into stages, each corresponding to a manageable unit of infrastructure advancement, a specific question, and a guaranteed publication opportunity.
+As PZCalibrate was indeed completed, it will be wrapped for incorporation into RAIL.
```
Comment (johannct) [inline README.md]: completed in what sense? Where is the code? Is the past tense a typo here?

**Review by johannct — APPROVED** [review]
LGTM, have a look at the 2 comments.

File: README.md (hunk 1 above)
Comment (aimalz) [reply README.md]: Done!

File: README.md (hunk 2 above)
Comment (aimalz) [reply README.md]: Good point, I thought The WiZZ was wrapped in there but seem to be mistaken? I took it out of the README. We should follow up with @morriscb about wrapping it for RAIL when the `Summarizer` superclass is ready.

**Pull request merged**